### PR TITLE
Add some missing changelog/upgrade infos

### PR DESCRIPTION
**Pull request by maidmaid**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony/pull/23387 https://github.com/symfony/symfony/pull/23376 https://github.com/symfony/symfony/pull/23363
| License       | MIT
| Doc PR        | /

Oops, some PRs have been merged without changelog/upgrade infos...

### CODE REVIEW

**Review by ogizanagi — APPROVED** [review]

File: src/Symfony/Component/Yaml/CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ CHANGELOG
    the parser and dumper is no longer supported, pass bitmask flags instead
  * the constructor arguments of the `Parser` class have been removed
  * the `Inline` class is internal and no longer part of the BC promise
+ * removed of the support for the `!str` tag, use the `!!str` tag instead
```
Comment (xabbuh) [inline src/Symfony/Component/Yaml/CHANGELOG.md]: removed support for the `!str` tag, use the `!!str` tag instead

File: src/Symfony/Component/Process/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ CHANGELOG
    methods of the `Process` class have been removed
  * support for passing `proc_open()` options has been removed
  * removed the `ProcessBuilder` class, use the `Process` class instead
+ * removed the `getEnhanceWindowsCompatibility()` and `setEnhanceWindowsCompatibility()` methods of `Process` class
```
Comment (xabbuh) [inline src/Symfony/Component/Process/CHANGELOG.md]: [...] of **the** `Process` [...]

File: src/Symfony/Component/Yaml/CHANGELOG.md (hunk 1 above)
Comment (maidmaid) [inline src/Symfony/Component/Yaml/CHANGELOG.md]: Updated

File: src/Symfony/Component/Process/CHANGELOG.md (hunk 2 above)
Comment (maidmaid) [inline src/Symfony/Component/Process/CHANGELOG.md]: Updated

File: src/Symfony/Component/Yaml/CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ CHANGELOG
    the parser and dumper is no longer supported, pass bitmask flags instead
  * the constructor arguments of the `Parser` class have been removed
  * the `Inline` class is internal and no longer part of the BC promise
+ * removed support for the !str tag, use the !!str tag instead
```
Comment (xabbuh) [inline src/Symfony/Component/Yaml/CHANGELOG.md]: I would still enclose the tags with backticks. My comment was meant to point out the missing article.

File: UPGRADE-4.0.md [hunk 4]
```diff
@@ -438,6 +438,8 @@ Process
 
  * Extending `Process::run()`, `Process::mustRun()` and `Process::restart()` is
    not supported anymore.
+   
+ * The `getEnhanceWindowsCompatibility()` and `setEnhanceWindowsCompatibility()` methods of `Process` class have been removed.
```
Comment (xabbuh) [inline UPGRADE-4.0.md]: missing "the" before `Process`

File: src/Symfony/Component/Yaml/CHANGELOG.md (hunk 3 above)
Comment (maidmaid) [inline src/Symfony/Component/Yaml/CHANGELOG.md]: Bad previous copy/past... It's fixed.

File: UPGRADE-4.0.md (hunk 4 above)
Comment (maidmaid) [inline UPGRADE-4.0.md]: Added.

**Review by xabbuh — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @maidmaid.

**Pull request merged**